ext/standard/basic_functions.stub.php

```diff
@@ -59,7 +59,7 @@ function krsort(array &$arg, int $sort_flags = SORT_REGULAR): bool {}
 
 function ksort(array &$arg, int $sort_flags = SORT_REGULAR): bool {}
 
-/** @param array|Countable $var */
+/** @param mixed $var */
 function count($var, int $mode = COUNT_NORAML): int {}
```

Comment:
While you're at it, could you please fix that `COUNT_NORAML` which should be `COUNT_NORMAL`?

Reply:
Sure! Nice catch!